tests/unit/OptionsListUtilsTest.js

```diff
@@ -221,26 +221,27 @@ describe('OptionsListUtils', () => {
         results = OptionsListUtils.getNewChatOptions(REPORTS, PERSONAL_DETAILS, 'peterparker@expensify.com');
 
         // Then one option will be returned and it will be the correct option
-        expect(results.personalDetails.length).toBe(1);
-        expect(results.personalDetails[0].text).toBe('Spider-Man');
+        expect(results.recentReports.length).toBe(1);
+        expect(results.recentReports[0].text).toBe('Spider-Man');
 
```

Comment:
We should test both instead of replacing the old `personalDetails` test

Reply:
Now I expect personalDetails to be empty

Reply:
> Now I expect personalDetails to be empty

Could we add that as a test case then?

Reply:
Yes, [here](https://github.com/Expensify/Expensify.cash/pull/2818/commits/49426e722f47bd458c3ea25bd6e6f95cf0126e02) it is.